ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/theme/ZashiColors.kt

```diff
@@ -0,0 +1,67 @@
+@file:Suppress("MagicNumber")
+
+package co.electriccoin.zcash.ui.design.theme
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+@Immutable
+data class ZashiColors(
```

Comment:
Let's file and link here a follow-up on extracting the color values to dedicated values so we can reuse them as we do in the original `Color.kt` approach

Reply:
done